### ISSUE TITLE
Blur on scroll 

### ISSUE THREAD

**Issue by howardyclo** [RRutsche/react-parallax#8]
Hello !  Here is a suggestion about blurring image when scrolling
I think this effect offers the better UX for users :-)
the effect is like this demo : http://codepen.io/sallar/pen/lobfp

Instead of setting constant blur value in setImagePosition()

```
if (this.props.blur) {
    this.img.style.WebkitFilter = 'blur(' + this.props.blur + 'px)';
    this.img.style.filter = 'blur(' + this.props.blur + 'px)';
}
```

I suggest that you can do something like this 

```
if (top > this.prevTop) // just detect user is scrolling top or down (or you can do it in more clever way)
    (this.blur >= 0 ) ? this.blur -= 1 * 0.05 : this.blur; // use 0.05 to prevent blurring too quick
else
    (this.blur <= this.props.maxBlurValue ) ? this.blur += 1 * 0.05 : this.blur;

this.prevTop = top; // save current top value for the next use
this.img.style.WebkitFilter = 'blur(' + this.blur + 'px)';
this.img.style.filter = 'blur(' + this.blur + 'px)';
```

Thanks for your project, this project helps a lot ! :-)

**Comment by rrutsche** [comment]
@HowardLoTW Thanks for your idea and for using the library. Looks indeed nice on the codepen page. Could imagine to add it to the project. I like the approach that behavior stays the same for everyone who already uses the blur feature. So maybe there could be a maxBlur and a minBlur value for this kind of dynamic blurring. The blur prop then would stay as it is.

**Comment by rcalabro** [comment]
Loved it! Hope you include in the project =)

**Comment by rrutsche** [comment]
@YuChunLOL @rcalabro 
Good things come to those who wait... hopefully. I just published version [1.2.5](https://www.npmjs.com/package/react-parallax) on npm with dynamic blur effect. You can now also pass an object as `blur` prop in format `{min: 0, max: 5}` - also negative values for min are possible to have better control over the position where blurring should start.
Just tell me if this is what you where thinking about and if it works for you. 
- best regards

**Comment by rrutsche** [comment]
I will close this issue for now. If there is any problem with the update please feel free to re-open the ticket
